practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/converter/FhirPractitionerConverter.java

```diff
@@ -24,13 +25,20 @@ public Practitioner convert(org.hl7.fhir.r4.model.Practitioner input) {
         if (input.getName().isEmpty()) {
             throw new IllegalArgumentException("FHIR Practitioner must have a name");
         }
-
+        
+        String idpIdentity = input.getIdentifier().stream()
+                                    .filter((id) -> id.getId().equals("IDP_IDENTITY"))
```

Comment:
What does IDP stands for? I'd use the full name if we're not short on chars...
Can we move this string to a const?